### PR TITLE
Use constant for services ports (to be used by tests lately)

### PR DESCRIPTION
**Pull request by tisnik**
As we are starting to make tests for various services, let's refactor theirs port numbers.

### CODE REVIEW

File: integration-tests/features/environment.py [hunk 1]
```diff
@@ -15,6 +15,10 @@
 # The following API endpoint is used to check if the system is started
 _API_ENDPOINT = 'api/v1/'
 
+# Ports used by various services
+_FABRIC8_ANALYTICS_SERVER = 32000
+_FABRIC8_ANALYTICS_JOBS = 34000
+_FABRIC8_MINIO_S3 = 33000
```
Comment (fridex) [inline integration-tests/features/environment.py]: What about parametrizing using env variables?

_FABRIC8_ANALYTICS_SERVER = os.environ.get('FABRIC8_ANALYTICS_SERVER', 32000)
_FABRIC8_ANALYTICS_JOBS = os.environ.get('FABRIC8_ANALYTICS_JOBS', 34000)
_FABRIC8_MINIO_S3 = os.environ.get('FABRIC8_MINIO_S3', 33000)

I'm not strictly against this change, just a suggestion for possible next steps. WDYT?

Comment (tisnik) [inline integration-tests/features/environment.py]: that's a good suggestion, tyvm. Could we do it in two steps then?

Comment (fridex) [inline integration-tests/features/environment.py]: Also probably remove "Minio" - we werer using different S3 local alternative but were forced to switch to Minio due to some features. Removing such names from variables can help in future when possible similar switch will be done.

Comment (fridex) [inline integration-tests/features/environment.py]: Yes, feel free to do it two steps. Thanks!

File: integration-tests/features/environment.py [hunk 2]
```diff
@@ -270,7 +274,7 @@ def before_all(context):
     
     context.coreapi_url = _add_slash(context.config.userdata.get(
         'coreapi_url',
-        'http://localhost:32000/'))
+        'http://localhost:' + str(_FABRIC8_ANALYTICS_SERVER) + '/'))
```
Comment (jpopelka) [inline integration-tests/features/environment.py]: or `'http://localhost:{port}/'.format(port=_FABRIC8_ANALYTICS_SERVER)`

Comment (tisnik) [inline integration-tests/features/environment.py]: even better, thx

**Pull request merged**